frontend/src/streamlist/__tests__/StreamList.test.jsx

```diff
@@ -27,22 +27,22 @@ afterEach(() => {
 });
 
 function verifyActionButtons(rowElement) {
-  const buttonsConatiner = rowElement.firstChild;
+  const buttonsContainer = rowElement.firstChild;
```

Comment:
conatiner

Reply:
`Conatiner` is the old version no?

Reply:
yeah i was just pointing it out cuz its funny, sorry